posthog/exceptions.py

```diff
@@ -43,6 +46,12 @@ def exception_reporting(exception: Exception, context: ExceptionContext) -> None
     Determines which exceptions to report and sends them to Sentry.
     Used through drf-exceptions-hog
     """
+    from posthog.api.instance_settings import get_instance_setting
+    from posthog.settings.base_variables import DEBUG
+
+    if DEBUG or get_instance_setting("CAPTURE_BACKEND_EXCEPTIONS"):
+        logger.exception(exception)
```

Comment:
How do log lines from this look like?

Reply:
```
2022-06-17T11:52:15.627710Z [error    ] faaack!                        [posthog.exceptions] ip=127.0.0.1 request_id=aa261442-54a3-42a2-b650-88eb5de6119a
Traceback (most recent call last):
  File "/Users/neilkakkar/Project/posthog/env/lib/python3.8/site-packages/rest_framework/views.py", line 506, in dispatch
    response = handler(request, *args, **kwargs)
  File "/Users/neilkakkar/Project/posthog/posthog/api/insight.py", line 545, in trend
    result = self.calculate_trends(request)
  File "/Users/neilkakkar/Project/posthog/posthog/decorators.py", line 52, in wrapper
    fresh_result_package = cast(T, f(self, request))
  File "/Users/neilkakkar/Project/posthog/posthog/api/insight.py", line 588, in calculate_trends
    result = trends_query.run(filter, team)
  File "/Users/neilkakkar/Project/posthog/posthog/queries/trends/trends.py", line 204, in run
    result.extend(handle_compare(filter, self._run_query, team, entity=entity))
  File "/Users/neilkakkar/Project/posthog/posthog/queries/base.py", line 51, in handle_compare
    trend_entity = func(filter=filter, team=team, **kwargs)
  File "/Users/neilkakkar/Project/posthog/posthog/queries/trends/trends.py", line 123, in _run_query
    raise Exception('faaack!')
Exception: faaack!
Internal Server Error: /api/projects/5/insights/trend/
Internal Server Error: /api/projects/5/insights/trend/
```

Reply:
Nitpicking: Could we add the endpoint to this log (e.g. as another argument after ip, etc)? I know it shows up below but with multiple parallel requests it might be hard to identify the endpoint/request.

Okay if this isn't in scope.